packages/js/src/plugins/auctionHouseModule/createBid.ts

```diff
@@ -163,19 +167,39 @@ export const createBidBuilder = async (
   };
 
   // Sell Instruction.
-  // ToDo: Add support for the auctioneerAuthority
   let buyInstruction;
-  if (tokenAccount) {
-    buyInstruction = createBuyInstruction({ ...accounts, tokenAccount }, args);
-  } else {
-    buyInstruction = createPublicBuyInstruction(
-      { ...accounts, tokenAccount: buyerTokenAccount },
-      args
+  if (params.auctioneerAuthority) {
+    const ahAuctioneerPda = findAuctioneerPda(
+      auctionHouse.address,
+      params.auctioneerAuthority.publicKey
     );
+
+    const buyAccounts = {
```

Comment:
Could we rename this so to something more accurate?

```suggestion
    const accountsWithAuctioneer = {
```